p2panda-js/README.md

```diff
@@ -108,15 +152,32 @@ const Chat = ({}) => {
 };
 ```
 
-## Development Setup
+### Manually load `.wasm`
```

Comment:
Very cool we offer this as well :+1:

And these multiple examples are great to have, very developer friendly!